great_expectations/datasource/fluent/sources.py

```diff
@@ -625,6 +636,11 @@ def __getattr__(self, attr_name: str):
             elif crud_method_type == CrudMethodType.ADD_OR_UPDATE:
                 return self.create_add_or_update_crud_method(datasource_type, docstring)
             elif crud_method_type == CrudMethodType.DELETE:
+                # deprecated-v0.17.2
+                warnings.warn(
+                    f"`{attr_name}` is deprecated as of v0.17.2 and will be removed in v0.19. Please use `delete_datasource` moving forward.",
```

Comment:
It says to use `delete_datasource` though your updated code shows usage of `delete`. Should this instead say to use `.sources.delete`?